packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -22,7 +22,7 @@ import { useLocation } from 'react-router-dom';
 
 import { RELEASE_SCHEMA } from '../../../shared/validation-schemas';
 import { pluginId } from '../pluginId';
-import { getTimezoneOffset } from '../utils/time';
+import { getTimezones } from '../utils/getTimezones';
 
```

Comment:
sorry to be annoying but I think getTimezones should be inside that utils/time, they're closely related